src/components/containers/seo.js

```diff
@@ -154,7 +154,7 @@ const SEO = ({ description, meta, title, no_index }) => {
                 type="text/javascript"
             ></script>
             <script>
-                {`window.location.hostname === 'www.deriv.com' && window.DD_RUM &&
+                {`(window.location.hostname === 'www.deriv.com' || window.location.hostname === 'deriv.com') && window.DD_RUM &&
```

Comment:
we dont use www.deriv.com hostname anymore, we resolved the hostname to always use `deriv.com`

Reply:
ok so we just need window.location.hostname === 'deriv.com' then?